pkg/kubelet/container/runtime.go

```diff
@@ -57,7 +57,10 @@ type Runtime interface {
 	Version() (Version, error)
 	// APIVersion returns the API version information of the container
 	// runtime. This may be different from the runtime engine's version.
+	// TODO(random-liu): We should fold this into Version()
 	APIVersion() (Version, error)
+	// CheckStatus checks whether the underlying runtime is working normally.
+	CheckStatus() error
```

Comment:
@vishh Good point. `Status()` is indeed more extensible, although it just returns error now. Will change it. :)